firestore/collgroupref.go

```diff
@@ -36,3 +46,146 @@ func newCollectionGroupRef(c *Client, dbPath, collectionID string) *CollectionGr
 		},
 	}
 }
+
+// TODO: add a fromProto to compliment toProto for query. With that serialization is
+// very possible. user could also do json.Marshal(toProto())
+
+// GetPartitionedQueries returns a slice of Query objects, each containing a
```

Comment:
Since all the other types/methods are now non-exported, this is the only thing that will go into the public package docs. Is there more to add here? Is there a link to the docs on cloud site for this feature?

Reply:
At the current point we don't have any docs really, other than rest docs, which I am not sure will bring clarity: https://firebase.google.com/docs/firestore/reference/rest/v1beta1/projects.databases.documents/partitionQuery